packages/medical-domain/src/domains/repositories/medical/medical-data.service.ts

```diff
@@ -355,11 +266,132 @@ class MedicalDataService {
         } else {
           message = String(error)
         }
-        console.debug({ message, rawData: raw })
+        console.log({ message, rawData: raw })
       }
     })
   }
 
+  add(data: MedicalData): void {
+    if (data.bolus) {
```

Comment:
How comes that we have to check that the value is defined? Doesn't the API returns an empty array instead of null?

Reply:
To be on the safe side ;)